compiler/one-cmds/tests/one-quantize_016.test

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+filename_ext="$(basename -- $0)"
+filename="${filename_ext%.*}"
+
+trap_err_onexit()
+{
+  echo "${filename_ext} FAILED"
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+inputfile="./reshape_matmul.circle"
+outputfile="./reshape_matmul.one-quantize_016.circle"
+
+rm -f ${filename}.log
+rm -f ${filename}.first.cdump
+rm -f ${filename}.second.cdump
+rm -f ${outputfile}
+
+if [[ ! -s "${inputfile}" ]]; then
+  one-import onnx -i reshape_matmul.onnx -o ${inputfile}
+fi
+
+# run test with different input_type
+one-quantize \
+--input_dtype float32 \
+--quantized_dtype uint8 \
+--granularity channel \
+--input_type uint8,int16 \
+--input_path ${inputfile} \
+--output_path ${outputfile} > ${filename}.log 2>&1
+
+if [[ ! -s "${outputfile}" ]]; then
+  trap_err_onexit
+fi
+
+circledump ${outputfile} | grep serving_default_l.1:0$ > ${filename}.first.cdump
+circledump ${outputfile} | grep serving_default_r.1:0$ > ${filename}.second.cdump
```

Comment:
Check dtype of a tensor using circledump

Reply:
I'm not sure `grep serving_default_r` name will stay with different versions of TF.

Reply:
alternative, 
```
circle-inspect --tensor_dtype quantized_conv2d.circle
```
-->
```
ifm UINT8
weights UINT8
bias INT32
ofm UINT8
```

Reply:
> alternative, but circle-inspect is not added to release package -_-;

Yeap. That' why I had to use `circledump`..